spark/dl/src/main/scala/com/intel/analytics/bigdl/models/inception/ImageNet2012.scala

```diff
@@ -72,14 +76,19 @@ object ImageNet2012Val {
      classNumber: Int
    )
    : DataSet[MiniBatch[Float]] = {
-     DataSet.SeqFileFolder.files(path, sc, classNumber).transform(
-       MTLabeledBGRImgToBatch[ByteRecord](
+
+     DataSet.SeqFileFolder.filesToImageFeatureDataset(path, sc, 1000).transform(
```

Comment:
above

Reply:
Refined, but this is specifically for ImageNet2012, not very common method